src/schema/typeDefs.ts

```diff
@@ -0,0 +1,68 @@
+const typeDefs: any = `
+type Query {
```

Comment:
What are the drawbacks of defining the schema in a typescript file instead of using a `schema.graphql` file?

Reply:
That I can't use graphql-tools to create typed schemas.